api/api.go

```diff
@@ -130,10 +131,108 @@ func (a *API) ExecuteTask(serviceID, taskKey string, inputData map[string]interf
 	if err = a.execDB.Save(exec); err != nil {
 		return "", err
 	}
-	go a.ps.Pub(exec, s.TaskSubscriptionChannel())
+	go a.ps.Pub(exec, s.ExecutionSubTopic())
 	return exec.ID, nil
 }
 
+// ListenEvent listens events matches with eventFilter on serviceID.
+func (a *API) ListenEvent(service string, f *EventFilter) (*EventListener, error) {
+	s, err := a.db.Get(service)
+	if err != nil {
+		return nil, err
+	}
+
+	if f != nil && f.Key != "" {
```

Comment:
what about `*`?

Reply:
When using command `service dev`, I get the error:
`⨯ Listening events error: rpc error: code = Unknown desc = Event "*" not found in service "Bitcoin"`

Reply:
done